n3fit/src/n3fit/model_gen.py

```diff
@@ -170,6 +171,10 @@ def observable_generator(
         boundary_condition: dict
             dictionary containing the instance of the a PDF set to be used as a
             Boundary Condition.
+        mask_array: np.ndarray
+            training mask per replica
+        validation_mask_array: np.ndarray
+            validation mask per replica, when not given ¬mask_array will be used
```

Comment:
Maybe it's worth clarifying (or citing where info can be found) in which scenario the validation mask can be different from ~training_mask